_data/faq.yml

```diff
@@ -140,7 +142,7 @@
       content: |
         If the server has ViaVersion and/or supports the latest Minecraft version, yes. However, we are unable to support older versions of Minecraft due to a limitation in our Java support library.
 
-    - title: Can I connect Geyser to a modded (Forge/Fabric) server?
+    - title: Can I connect Geyser to a modded (Forge/Fabric/NeoForge) server?
```

Comment:
Are we replacing Forge or adding to it?

Reply:
We're adding Geyser-NeoForge, but this faq question applies to all modded servers